clients/python-wrapper/lakefs/object.py

```diff
@@ -665,13 +679,28 @@ def writer(self,
         lakeFS. The optional parameters can be modified by accessing the respective fields as long as the writer is
         still open.
 
+        Usage example of reading a file from local file system and writing it to lakeFS:
+
+        .. code-block:: python
+
+            import lakefs
+
+            obj = lakefs.repository("<repository_name>").branch("<branch_name>").object("my_image")
+
+            with open("my_local_image", mode='rb') as reader:
+                with obj.writer("wb") as writer:
+                    writer.write(reader.read())
```

Comment:
I think you can chain with statements:

```suggestion
            with open("my_local_image", mode='rb') as reader, obj.writer("wb") as writer:
                writer.write(reader.read())
```